internal/fsutil/create.go

```diff
@@ -121,6 +125,28 @@ func createSymlink(o *CreateOptions) error {
 	return os.Symlink(o.Link, o.Path)
 }
 
+func createHardLink(o *CreateOptions) error {
```

Comment:
We need tests for this in `create_tests.go`.

Reply:
Added in [85a4cf9](https://github.com/letFunny/chisel/pull/17/commits/85a4cf90cc99950c89fafb834518cf97f07b6a56), [19fc4ff](https://github.com/letFunny/chisel/pull/17/commits/19fc4ff868e8bfd327c3ff8a4635ef4c8fc76070) and [eefd744](https://github.com/letFunny/chisel/pull/17/commits/eefd7447c843615120cddcfd890c511a777e2cf3).